UefiCpuPkg/Library/CpuExceptionHandlerLib/DxeException.c

```diff
@@ -179,9 +181,23 @@ InitializeCpuInterruptHandlers (
                   EFI_SIZE_TO_PAGES (TemplateMap.ExceptionStubHeaderSize * CPU_INTERRUPT_NUM),
                   (EFI_PHYSICAL_ADDRESS *)(UINTN)&InterruptEntryCode
                   );
-  // MU_CHANGE END
   ASSERT (!EFI_ERROR (Status) && InterruptEntryCode != NULL);
 
+  ASSERT_EFI_ERROR (gBS->LocateProtocol (&gMemoryProtectionSpecialRegionProtocolGuid, NULL, (VOID **)&SpecialRegionProtocol));

```

Comment:
You'll have to move this code outside the assert.

Might not execute if debug asserts are disabled.

Reply:
Ah yes my mistake